common/app/model/Cached.scala

```diff
@@ -146,6 +145,8 @@ object NoCache {
 
 case class NoCache[A](action: Action[A]) extends Action[A] {
 
+  implicit val ec: ExecutionContext = ExecutionContexts.executionContext
```

Comment:
I think the `case class` should receive an implicit execution context rather than having it as an attribute. Chances are that every controller that uses `NoCache` already have an execution context somewhere.